packages/apps/human-app/frontend/src/modules/worker/jobs/hooks/use-filter-updates.ts

```diff
@@ -0,0 +1,23 @@
+import { useJobsFilterStore } from './use-jobs-filter-store';
```

Comment:
Can't drop a comment in the corresponding file, so will leave it here: ;et's be consistent and also have `useJobsTypesOraclesFilterStore` instead of `useJobsTypesOraclesFilter` (check other "stores" as well please)